clients/web/src/mocks/scenarios/org_assessment_multiple.js

```diff
@@ -0,0 +1,67 @@
+import { initIncompleteAssessment, initCompleteAssessment } from '../utils'
+
+export default function (
+  server,
+  orgId,
+  orgName = 'Organization With Multiple Assessments'
+) {
+  const current_construction = server.create('assessment', {
+    account: orgId,
+    industryName: 'Construction',
+    industryPath: '/construction/',
+  })
+
+  const current_enclosures = server.create('assessment', {
+    account: orgId,
+    industryName: 'Boxes & enclosures',
+    industryPath: '/metal/boxes-and-enclosures/',
```

Comment:
On the API side, there is no trailing '/' in paths (ex: `/metal/boxes-and-enclosures`)

Reply:
👌 Must have been thrown off by some of the sample responses e.g. https://www.tspproject.org/docs/api/#CreatePageElementEditableDetail. Ok, I'll change it. Does this mean that *all* API calls do not have a trailing slash?

Reply:
Yes, per our meeting, the trailing slash is not necessary in API calls. I will create card for this.